client_sdk/go/fpc/policy/policy.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package policy
+
+import (
+	cb "github.com/hyperledger/fabric-protos-go/common"
+	"github.com/hyperledger/fabric-protos-go/msp"
+	"github.com/hyperledger/fabric/common/policydsl"
+)
+
+// MajorityOfMembers returns an endorsement policy that requires the signatures
+// by a majority of the given members.
+//
+// Note that this endorsement policy is not flexible enough to support membership
+// changes. A more robust majority-based endorsement policy can be provided
+// through the channels default endorsement policy.
```

Comment:
I think this should be a stronger message given the potential insecurity!   

BTW: i hope we are actually also allowing _not_ specyfing a policy so the default policy applies (another advantage, besides the inherent robustness, of my proposal to drop this and just use in both test-cases the default policy ... ;-))

```suggestion
//
// IMPORTANT: Note that this endorsement policy is not flexible enough to support membership changes and _might become unintendently *insecure* when membership increases. 
// A more robust majority-based endorsement policy can be defined 
// by not specifying an explicit policy and hence applying the channels default endorsement policy (which is majority in the default fabric network configurations).
```